providers/fcontext.rb

```diff
@@ -25,6 +25,23 @@ def restorecon(file_spec)
   return "restorecon -iR #{path}"
 end
 
+def semanage_options(filetype)
+  # Set options for filetype
+  if filetype
```

Comment:
I am not 100% sure, so I would not use it as default. To be honest, I'm not 100% sure -l a is exactly the same as not using the -l option, that's why I'd default to not using the option if nothing's specified in the filetype parameter for the resource.